Added DocBook snippets as a separate extension (#9)
openSUSE/suse-vscode-doc: extensions/docbook-snippets/CHANGELOG.md

# Change Log

> All notable changes to the "docbook-snippets" extension will be documented in this file.

Reviewer: ```suggestion
All notable changes to the "docbook-snippets" extension are documented in this file.
```
